auth/build.gradle.kts

```diff
@@ -1,3 +1,6 @@
+
+import Config.Libs.Lint.api
+import Config.Plugins.android
```

Comment:
Mmmm, I'd just get rid of these from GitHub. Looks like AS keeps adding them back for some reason.

Reply:
:cry:

Reply:
Let's not worry about it, I'd rather not fight AS esp when this isn't code we ship to developers.